blinka/commands/font.py

```diff
@@ -55,22 +55,37 @@ def make_suggested_filename(font, size):
 def output_header(file, bounding_box):
     file.write("STARTFONT 2.1\n")
     file.write("FONTBOUNDINGBOX %d %d %d %d\n" % (bounding_box[0], bounding_box[1], bounding_box[2], bounding_box[3]))
+
+    properties = {}
+    properties['FONT_ASCENT'] = bounding_box[4]
+    properties['FONT_DESCENT'] = bounding_box[5]
+    if len(properties.keys()) > 0:
+        # Output properties
+        file.write("STARTPROPERTIES %d\n" % len(properties.keys()))
+        for key in properties:
+            file.write("%s %s\n" % (key, properties[key]))
+        file.write("ENDPROPERTIES\n")
+
     file.write("CHARS\n")
 
 def output_footer(file):
     file.write("ENDFONT\n")
 
 def update_bounding(bounding_box, glyph):
     if not bounding_box:
-        bounding_box = [0, 0, 0, 0]
+        bounding_box = [0, 0, 0, 0, 0, 0]
     x = glyph.bitmap.width
     y = glyph.bitmap.height
     x_offset = 0
     y_offset = -glyph.descent if glyph.descent else glyph.ascent - glyph.height
+    ascent = glyph.ascent
```

Comment:
why not move this declaration up so you can use it in your y_offset calculation?

Reply:
Ignorance. But in fairness it made the diffs cleaner.

Reply:
makes sense. shippit